src/lib/concurrency/transaction_manager.hpp

```diff
@@ -82,7 +82,13 @@ class TransactionManager : private Noncopyable {
 
  private:
   std::atomic<TransactionID> _next_transaction_id;
+  // TransactionID = 0 means "not set" in the MVCC columns
+  static constexpr auto initial_transaction_id = TransactionID{1};
+
   std::atomic<CommitID> _last_commit_id;
+  // We use commit_id=0 for rows that were inserted and then rolled back. Also, this can be used for rows that have
+  // been there "from the beginning of time".
+  static constexpr auto initial_commit_id = CommitID{1};
```

Comment:
Shouldn’t these be in `types.hpp` and all caps like the other constants?

Reply:
I wouldn't say so. Those are internal to the tx manager and should not be exposed.

Reply:
okay, and all caps?

Reply:
done